src/main/java/org/openmaptiles/layers/Transportation.java

```diff
@@ -296,11 +340,7 @@ public List<OsmRelationInfo> preprocessOsmRelation(OsmElement.Relation relation)
       String colour = coalesce(
         nullIfEmpty(relation.getString("colour")), nullIfEmpty(relation.getString("ref:colour")));
 
-      if ("e-road".equals(network)) {
-        networkType = RouteNetwork.E_ROAD;
-      } else if ("AsianHighway".equals(network)) {
-        networkType = RouteNetwork.A_ROAD;
-      } else if ("US:I".equals(network)) {
```

Comment:
did we mean to remove these? If so we should remove them from the enum list or otherwise keep them here.